_people/iona_hampson.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Iona Hampson
+headshot: 7bg8jk3
+gender: female
+graduated: 2017
+course:
+  - Ancient History and History BA
+  - Broadcast Journalism MA
```

Comment:
Can the second course be added as a comment instead of the course field. It's not one undertaken at this Uni but is still data to be kept :)

Reply:
@johnathan99j :)